### PR TITLE
Move to IBootstrap

### PR DESCRIPTION
**Pull request by juliushaertl**

### CODE REVIEW

File: lib/Mount/GroupFolderStorage.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function getOwner($path) {
 	public function instanceOfStorage($class) {
 		// "implement" the interface without adding a hard dependency on nc15
 		if ($class === 'OCP\Files\Storage\IDisableEncryptionStorage') {
-			return true;
+			return false;
```
Comment (ChristophWurst) [inline lib/Mount/GroupFolderStorage.php:68]: why?

Comment (juliushaertl) [reply lib/Mount/GroupFolderStorage.php:68]: Woops not sure where that came from

Comment (juliushaertl) [reply lib/Mount/GroupFolderStorage.php]: Fixed.

**Comment by juliushaertl** [comment]
Tests will be fixed with https://github.com/nextcloud/groupfolders/pull/1188

**Review by ChristophWurst — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**